pyaedt/generic/configurations.py

```diff
@@ -0,0 +1,1278 @@
+import copy
+import json
+import os
+from collections import OrderedDict
+from datetime import datetime
+
+from pyaedt import __version__
+from pyaedt.generic.general_methods import _create_json_file
+from pyaedt.generic.general_methods import generate_unique_name
+from pyaedt.generic.general_methods import pyaedt_function_handler
+from pyaedt.modeler.GeometryOperators import GeometryOperators
+from pyaedt.modeler.Modeler import CoordinateSystem
+from pyaedt.modules.Boundary import BoundaryObject
+from pyaedt.modules.DesignXPloration import SetupOpti
+from pyaedt.modules.DesignXPloration import SetupParam
+from pyaedt.modules.MaterialLib import Material
+from pyaedt.modules.Mesh import MeshOperation
+
+
+def _find_datasets(d, out_list):
+    for v in list(d.values()):
+        if isinstance(v, (dict, OrderedDict)):
+            _find_datasets(v, out_list)
+        else:
+            a = copy.deepcopy(v)
+            val = a
+            if str(type(a)) == r"<type 'List'>":
+                val = list(a)
+            if isinstance(val, list):
+                for el in val:
+                    try:
+                        if "pwl" in el["free_form_value"]:
+                            out_list.append(
+                                el["free_form_value"][el["free_form_value"].find("$") : el["free_form_value"].find(",")]
+                            )
+                    except KeyError:
+                        pass
+            elif isinstance(val, str):
+                if "pwl" in val:
+                    out_list.append(val[val.find("$") : val.find(",")])
+
+
+class ConfigurationsOptions(object):
+    """Options class for the configurations.
+    User can enable or disable import export components."""
+
+    def __init__(self):
+        self._object_mapping_tolerance = 1e-9
+        self._export_variables = True
+        self._export_setups = True
+        self._export_optimizations = True
+        self._export_parametrics = True
+        self._export_boundaries = True
+        self._export_mesh_operations = True
+        self._export_coordinate_systems = True
+        # self._export_face_coordinate_systems = False
+        self._export_materials = True
+        self._export_object_properties = True
+        self._import_variables = True
+        self._import_setups = True
+        self._import_optimizations = True
+        self._import_parametrics = True
+        self._import_boundaries = True
+        self._import_mesh_operations = True
+        self._import_coordinate_systems = True
+        # self._import_face_coordinate_systems = False
```

Comment:
```suggestion
```

Reply:
Face coordinate systems support will be implemented in a second stage. We started implementing it, but it is not finished yet.

Reply:
ok, thanks @Alberto-DM for the update!